src/clr-icons/clr-icons.spec.ts

```diff
@@ -5,32 +5,49 @@
  */
 import {itIgnore} from "../../tests/tests.helpers";
 
+import {ClarityIcons} from ".";
```

Comment:
Why has the import changed from `./index` to `.`

Reply:
I used VScode plugin that organizes imports and it also turned `"./index"` to `"."`. But I can revert it back.